src/routes/home/HomeView.vue

```diff
@@ -198,21 +198,29 @@ export default {
     },
   },
   watch: {
+    tab(newVal){
+      this.setGameListUrl(newVal);
+    },
     $route: {
       immediate: true,
       handler() {
         if (this.$route.query?.error) {
           this.handleSubscribeError(Number(this.$route.query.gameId), this.t(this.$route.query.error));
-          this.$router.replace('/');
+          this.$router.push('/');
```

Comment:
This can stay replace
```suggestion
          this.$router.replace('/');
```